test/fixtures/dialects/snowflake/snowflake_alter_schema.sql

```diff
@@ -0,0 +1,4 @@
+alter schema if exists schema1 rename to schema2;
+alter schema schema1 swap with schema2;
+alter schema schema2 enable managed access;
+alter schema schema1 set data_retention_time_in_days = 3;
```

Comment:
Can you add more tests? At least one use of unset, and ideally set & unset delimited tags

Reply:
Good idea, I've added those tests.